draft-ietf-quic-transport.md

```diff
@@ -736,6 +733,18 @@ and will contain a CRYPTO frame with an offset matching the size of the CRYPTO
 frame sent in the first Initial packet.  Cryptographic handshake messages
 subsequent to the first do not need to fit within a single UDP datagram.
 
+### Handling of Fatal Initial Packets
+
+The contents of some Initial packets may, according to this specification, force
```

Comment:
I'm not a big fan of a specification saying you can ignore the specification; I'd prefer consistency.  You removed the language above indicating that things should be fatal, so you're part-way there.

Perhaps something like "Receipt of an invalid Initial packet could indicate an injection attack rather than a peer failure.  Endpoints abort a connection attempt if no valid Initial packets are received after an appropriate timeout."  That gives enough latitude that the timeout could be immediate or a long observation for multiple possibilities.

Reply:
I really don't want to go everywhere it says we MUST terminate the connection and then add "unless it's an Initial and ..." Inevitably it's a bit clunky.

Your language is fine, but I'd like to preserve the option to just drop the packet without running any timer. It's a simpler course of action for the endpoint and I don't see a significant downside.